src/main/java/com/jcabi/log/ColorfullyFormatted.java

```diff
@@ -63,8 +63,8 @@ public ColorfullyFormatted(final String bas, final String col) {
      */
     @Override
     public String format() {
-        return this.basic.replace(
-            new ControlSequenceIndicatorFormatted("%s?m").format(),
+        return this.basic.replaceAll(
+            new ControlSequenceIndicatorFormatted("%s\\?m").format(),
```

Comment:
@amihaiemil I see, however such notes should go to the PR description, not to the comments. But thanks for adding them, they are useful.